src/Frontend/Panes/Lobbies/CreatePlanetPane.tsx

```diff
@@ -0,0 +1,170 @@
+  DarkForestNumberInput,
+  NumberInput,
+} from '../../Components/Input';
+import { Row } from '../../Components/Row';
+import { Green, Red, Sub, Subber } from '../../Components/Text';
+import { Btn } from '../../Components/Btn';
+import { Table } from '../../Views/Table';
+
+import { LobbiesPaneProps, Warning } from './LobbiesUtils';
+import styled from 'styled-components';
+import { AdminPlanet } from '@darkforest_eth/types';
+import _ from 'lodash';
+
+const rowChunkSize = 4;
+const rowStyle = { gap: '8px' } as CSSStyleDeclaration & React.CSSProperties;
+// Handling the non-input lvl 0 by calculating the items in the row
+const itemStyle = { flex: `1 1 ${Math.floor(100 / rowChunkSize)}%` };
+
+const TableContainer = styled.div`
+  overflow-y: scroll;
+  width: 100%;
+`;
+
+function formatBool(bool: boolean) {
+  return bool ? <Green>Y</Green> : <Red>N</Red>;
+}
+
+const defaultPlanet: AdminPlanet = {
+  x: 0,
+  y: 0,
+  level: 0,
+  planetType: 0,
+  requireValidLocationId: false,
+  revealLocation: true,
+  isTargetPlanet: false,
+  isSpawnPlanet: false,
+};
+
+export function CreatePlanetPane({ config: config, onUpdate: onUpdate }: LobbiesPaneProps) {
+  const [planet, setPlanet] = useState<AdminPlanet>(defaultPlanet);
+
+  const headers = ['Coords', 'Level', 'Type', 'Require Valid', 'Reveal', 'Target', 'Spawn', ''];
+  const alignments: Array<'r' | 'c' | 'l'> = ['c', 'c', 'c', 'c', 'l', 'l', 'l', 'l'];
+  const columns = [
+    (planet: AdminPlanet) => (
+      <Sub>
+        ({planet.x}, {planet.y})
+      </Sub>
+    ),
+    (planet: AdminPlanet) => <Sub>{planet.level}</Sub>,
+    (planet: AdminPlanet) => <Sub>{planet.planetType}</Sub>,
+    (planet: AdminPlanet) => formatBool(planet.requireValidLocationId),
+    (planet: AdminPlanet) => formatBool(planet.revealLocation),
+    (planet: AdminPlanet) => formatBool(planet.isTargetPlanet),
+    (planet: AdminPlanet) => formatBool(planet.isSpawnPlanet),
+    (planet: AdminPlanet, i: number) => (
+      <Btn onClick={() => onUpdate({ type: 'ADMIN_PLANETS', value: planet, index: i })}>X</Btn>
+    ),
+  ];
+
+  function AdminPlanets({ config }: LobbiesPaneProps) {
+    const adminPlanets = config.ADMIN_PLANETS.currentValue;
+    return adminPlanets && adminPlanets.length > 0 ? (
+      <TableContainer>
+        <Table
+          paginated={true}
+          rows={adminPlanets || []}
+          headers={headers}
+          columns={columns}
+          alignments={alignments}
+        />
+      </TableContainer>
+    ) : (
+      <Sub>No planets created</Sub>
+    );
+  }
+
+  function Tips() {
+    return (
+      <>
+        <Row>
+          <span>Tips about planet creation</span>
+        </Row>
+        <Subber>
+          <Row>
+            Planet types: 0-Planet, 1-Asteroid, 2-Foundry, 3-Spacetime Rip, 4-Phasar
+          </Row>
+        </Subber>
+        <Row>
+          <Subber>
+            If you want to play with spawn planets activated, you must create at least one revealed Spawn
```

Comment:
It seems easy enough to check that if spawn planets is true, you must be creating at least 1 planet with `isSpawnPlanet` checked

Reply:
probably in `CreateLobby.tsx`

Reply:
Problem here is order of operations -> 1. create lobby 2. create spawn planets. So there is no way to validate that a spawn planet has been created before allowing a player to create a lobby.
Moving forward, I'm going to divide lobby creation into two steps:
1. Customize lobby config and launch lobby
2. Add planets to world and players to whitelist

With this system, we can have proper validation that the lobby creator has added spawn planets to the world before launching the lobby from step 2. They could still enter the world if they access the proper url, but it should be a large enough barrier to ensure the average player isnt being daft